src/js/components/Sort.js

```diff
@@ -40,10 +39,13 @@ export default class Sort extends Component {
         responsive={false}>
         <Select value={label} options={options} onChange={this._onChange} />
         <Box direction='row' flex={false} responsive={false} align='center'>
-          <Button icon={<AscIcon />}
+          <Button
+            icon={<AscIcon
+                colorIndex={props.direction === 'asc' ? 'brand' : null} />}
```

Comment:
can we move direction to line 31 ?

also can we replace `null` with `undefined` ?